src/helpers/translations/translations.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  locale: "en",
+  crypto: {
+    ethereum: "etherereum",
+    tokens: "tokens",
+    nft: "Non-Fungible Token",
+    whatIsANFT:
+      " is a way to ensure digital assets are unique and easily tradable on a blockchain.",
+    harbergerTax: "Harberger Tax"
+  },
+  muggle: {
+    ethereum: "blockchain",
+    tokens: "animals",
+    nft: "Crypto Collectable",
+    whatIsANFT:
+      " is a digital asset that is completely unique and easily tradable on a blockchain.",
+    harbergerTax: "Monthly Pledge"
```

Comment:
GLitchey I cant comment on lines above. This is my comment for whatIsANFT. "is a unique digital trading card that can only have one owner" is a suggestion. Think there are other options too.

Reply:
" is a digital animal that is completely unique and easily tradable on a blockchain."
This felt best after changing crypto collectable to blockchain animal

Reply:
I resolved some conversations above which I'm not sure if you can see the replies above